packages/charts/src/hooks/useRenderHighlightedCells.ts

```diff
@@ -0,0 +1,13 @@
+import { getValueByDataKey } from 'recharts/lib/util/ChartUtils';
```

Comment:
this file isn't used and can be deleted?

Reply:
the function is used in line 9